scss/_utilities.scss

```diff
@@ -88,9 +88,19 @@ $utilities: map-merge(
         sm: var(--#{$prefix}box-shadow-sm),
         lg: var(--#{$prefix}box-shadow-lg),
         none: none,
```

Comment:
If we set `$enable-bootstrap-compatibility: true`, we have two `.shadow-none` generated. It might worth removing this line so that the one generated is always coming from OUDS in priority, right?